augur/application/schema/alembic/versions/1_augur_new_changes.py

```diff
@@ -35,12 +36,14 @@ def upgrade():
    add_user_repo_table_12()
    add_materialized_views_13()
    set_repo_name_path_null_14()
+   convert_repo_deps_libyear_dates_15()
```

Comment:
This is a serialization and deserialization of all schema changes over time. Current instances will break or be inconsistent and broken if we add things mid-construction like this; It'll be ok on any new instances, but problematic for existing ones.